cola/widgets/diff.py

```diff
@@ -1345,7 +1345,7 @@ def revert_selection(self, *, edit=False):
                     'This operation drops uncommitted changes.\n'
                     'These changes cannot be recovered.'
                 ),
-                N_('Revert the uncommitted changes?'),
+                N_('Revert uncommitted changes from the selected lines?'),
```

Comment:
For some reason I failed to note that the button text reflects this already. And it's likely wrong given the above conditional anyway. So nothing to see here, moving along and closing.